runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutor.java

```diff
@@ -83,6 +86,25 @@
         objectMapper = new ObjectMapper(factory);
     }
 
+    static {
+        final ThreadFactory threadFactory = new ThreadFactoryBuilder()
+                .setNameFormat("python-engine-%d")
+                .setDaemon(true)
+                .build();
+        engineExecutorService = Executors.newFixedThreadPool(10, threadFactory);
+
+    }
+
+    static {
+        final ThreadFactory threadFactory = new ThreadFactoryBuilder()
+                .setNameFormat("python-test-%d")
+                .setDaemon(true)
+                .build();
+        testExecutorService = Executors.newFixedThreadPool(3, threadFactory);
```

Comment:
3 needs to be a exposed as a system property with default

Reply:
done